README.md

```diff
@@ -165,8 +172,12 @@ For `alt-da` clients running on Optimism, the following commitment schema is sup
   type       type     byte
 ```
 
-### Generic Commitment Mode
-For generic clients communicating with proxy, the following commitment schema is supported:
+Both `keccak256` (i.e, S3 storage using hash of pre-image for commitment value) and `generic` (i.e, EigenDA) are supported to ensure cross-compatibility with alt-da storage backends if desired by a rollup operator.
```

Comment:
"hash of pre-image for commitment value" sounds weird, since hash of pre-image is just the commitment value itself.
Can we say something like "using hash of the value as the S3 key" or something like that?

Reply:
maybe I'm missing something as well.

Reply:
they're both pretty idempotent